content/en/user-manual/publishing/playable-ads/snapchat-playable-ads.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Snapchat Playable Ad
+template: usermanual-page.tmpl.html
+position: 6
+---
+
+PlayCanvas supports the Snapchat Playable Ad format and requirements via an [official external tool on GitHub][2].
+
+Snapchat playable ads uses [MRAID 2.0 API][mraid-api] standard and requires the assets for the ad to be external from the `index.html` rather than embedded.
```

Comment:
```suggestion
Snapchat playable ads uses the [MRAID 2.0 API][mraid-api] standard and requires the assets for the ad to be external from the `index.html` rather than embedded.
```